akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java

```diff
@@ -94,7 +97,7 @@ public DoLog(String text) {
           // note that since registration of workers goes through the receptionist there is no
           // guarantee the router has seen any workers yet if we hit it directly like this and
           // these messages may end up in dead letters - in a real application you would not use
-          // a group router like this - it is to keep the sample simple
+          // a group router immediately like this - it is to keep the sample simple
```

Comment:
This comment is actually not true anymore, a group router now stash until it sees the first listing, I'll create an issue about that